text/3455-t-test.md

```diff
@@ -0,0 +1,110 @@
+Simultaneously, there are a number of well known needs and feature requests in the space that do not have an explicit
+owner driving the efforts to completion.
+
+For example, there's been some long standing requests to have additional test output
+formats, such as JSON ([#49359]) and JUnit, available on Stable Rust.
+While some of these are available as unstable features in Nightly Rust,
+in [Rust 1.70.0](https://blog.rust-lang.org/2023/06/01/Rust-1.70.0.html), a bug was fixed
+so unstable test features, like `cargo test -- --format json`, require using a nightly
+toolchain, like other unstable features. This caused breakage in certain editor/IDE and CI
+related tooling as they had been relying on the prior behavior to get test data
+programmatically, and little progress has been made in the last 5 years ([#49359]).
+
+Furthermore, there's been a growing interest in improving testing generally, like [better CI integration][ci] as well as requests for things like better support for custom test harnesses and frameworks (.e.g [#2318]).
+
+The new Testing team is intended to establish an overarching vision and provide focused attention on these areas. 
+
+[#49359]: https://github.com/rust-lang/rust/issues/49359
+[#50297]: https://github.com/rust-lang/rust/issues/50297
+[#2318]: https://github.com/rust-lang/rfcs/pull/2318
+[ci]: (https://internals.rust-lang.org/t/pre-rfc-implementing-test-binary-list-format-json-for-use-by-ide-test-explorers-runners/18308)
+
+## Mission and responsibilities
+[mission]: #mission
+
+This team would be primarily focused on iterating on the test writing and analysis experience, `cargo test`, and enabling integration points and features for external tools like CI or IDEs.
+
+Examples of issues to resolve:
+- [Stabilize support for programmatic (json) test output](https://github.com/rust-lang/rust/issues/49359)
+- What parts of [cargo nextest](https://nexte.st/) can we stabilize?
+- With the proliferation of test frameworks (e.g. [rstest], [trybuild], [trycmd], [cargo_test_support], [criterion]), are there underlying needs we can resolve?
+
+## Relationships to other teams
+
+With the aforementioned breadth across the Project, the Testing team will need to have collaborative relationships with many other teams, and is conceptually a subteam of both the Libs and Dev Tools teams.
+
+The rust-lang/team repo does not currently support representing dual-parent subteams, so for now the Testing team will be primarily under the Dev Tools team
+
+T-devtools: This will be the primary top level parent team.
+
+**T-cargo**: This is a sibling team that T-testing will need to work with similarly to T-rustfmt, T-clippy, etc.
+
+**T-rustdoc**: This is a sibling team that T-testing will likely coordinate with if any changes are need to how we do doctesting
+
+**T-IDEs and Editors**: This is a sibling team that T-testing will likely coordinate with to understand the needs of IDEs/editors related to incorporating test related capabilities
```

Comment:
When I initially added this sibling team relationship I had RA and the rest of the editor/IDE landscape community in mind. However, I'm now remembering that RA is a separate team altogether, and I'm actually a little unclear on whether T-ides-and-editors is still a functionally active team?

Obviously the items captured in this list are not an exhaustive, binding list (we'll work with other teams as and when necessary regardless) but if anyone knows offhand whether this team makes sense and/or has thoughts on whether we should mention the RA team specifically that'd be good to know

Reply:
it's not an active team, we should remove this and instead work with the r-a team here